kairon/actions/definitions/custom_parallel_actions.py

```
@@ -0,0 +1,203 @@
+
+        @param action_name: The name of the action.
+        @param action_instance: The instance of the action to execute.
+        @return: The response from the webhook call.
+        """
+        request_json = action_instance
+        request_json['next_action'] = action_name
+
+        url = Utility.environment["action"].get("url")
+
+        async with aiohttp.ClientSession() as session:
+            async with session.post(url, json=request_json) as response:
```

Ruff (0.11.9)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)